### PR TITLE
Run the new upstreamize.rb on the Managing Hosts guide

### PR DESCRIPTION
**Pull request by spetrosi**
Bug 1835683 - [upstream] Move the Managing Hosts upstream

https://bugzilla.redhat.com/show_bug.cgi?id=1835683

### CODE REVIEW

**Comment by spetrosi** [comment]
@lzap updated the upstreamize.rb script in #184, in this PR I am running the script to replace some non-upstreamized lines with attributes in the Managing Hosts guide.

**Comment by tahliar** [comment]
This time while scrolling through a Meld diff I spotted some commands in proc_registering-a-host-to-satellite-using-the-bootstrap-script.adoc that use satellite6.example.com

File: guides/doc-Managing_Hosts/topics/proc_integrating_satellite_with_red_hat_web_console.adoc [hunk 1]
```diff
@@ -5,8 +5,8 @@ By default, Red Hat web console integration is disabled in {Project}. If you wan
 
 .Procedure
 
-* On {ProjectServer}, run `satellite-installer` with the  `--enable-foreman-plugin-remote-execution-cockpit` option:
+* On {ProjectServer}, run `{foreman-installer}` with the  `--enable-foreman-plugin-remote-execution-cockpit` option:
 +
 ----
-# satellite-installer --enable-foreman-plugin-remote-execution-cockpit
+# {foreman-installer} --enable-foreman-plugin-remote-execution-cockpit
```
Comment (tahliar) [inline guides/doc-Managing_Hosts/topics/proc_integrating_satellite_with_red_hat_web_console.adoc:11]: This one isn't rendering - the part of the script that adds "attributes" to subs= at the top must have missed this one as it doesn't have any options included at all.

File: guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc [hunk 2]
```diff
@@ -13,7 +13,7 @@ This section details the workflow for:
 . To enable the plug-in on your {ProjectServer}, enter the following command:
 +
 ----
-# satellite-installer --enable-foreman-plugin-templates
+# {foreman-installer} --enable-foreman-plugin-templates
```
Comment (tahliar) [inline guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc:16]: This one also doesn't render due to a missing options header.

File: guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc [hunk 3]
```diff
@@ -292,7 +292,7 @@ You can override default API settings by specifying them in the request with the
 $ curl -H "Accept:application/json,version=2" \
 -H "Content-Type:application/json" \
 -u login:password \
--k https://satellite.example.com/api/v2/templates/export \
+-k https://{foreman-example-com}/api/v2/templates/export \
```
Comment (tahliar) [inline guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc:295]: Doesn't render.

File: guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc [hunk 4]
```diff
@@ -320,7 +320,7 @@ To avoid errors after removing the foreman_templates plugin:
 . Disable the plug-in using the {Project} installer:
 +
 ----
-# satellite-installer --no-enable-foreman-plugin-templates
+# {foreman-installer} --no-enable-foreman-plugin-templates
```
Comment (tahliar) [inline guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc:323]: Doesn't render.

**Comment by spetrosi** [comment]
> This time while scrolling through a Meld diff I spotted some commands in proc_registering-a-host-to-satellite-using-the-bootstrap-script.adoc that use satellite6.example.com

Thank you for the thorough review, I must have checked via meld myself. Non-working attributes and instances of satellite.example.com are now fixed.

**Comment by tahliar** [comment]
> > This time while scrolling through a Meld diff I spotted some commands in proc_registering-a-host-to-satellite-using-the-bootstrap-script.adoc that use satellite6.example.com
> 
> Thank you for the thorough review, I must have checked via meld myself. Non-working attributes and instances of satellite.example.com are now fixed.

No worries, they weren't actually highlighted, I just happened to slow my scroll and see them. Would have been easy to miss! :scream:

**Pull request merged**